.gitignore

```diff
@@ -7,3 +7,4 @@ __pycache__
 .eggs/
 test-report.xml
 venv/
+.idea/
```

Comment:
Sorry, but this needs to go. We try to only put things in .gitignore that a) exist in everyone's working copy or b) are created by the build.

http://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file#answer-5724476